### PR TITLE
fix server_jupyter

### PR DESCRIPTION
**Pull request by joamatab**
fix server Jupyter https://github.com/gdsfactory/kweb/issues/32

i had some issues testing this with gdsfactory, so may not be completely fixed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@ __pycache__/
 extra/
 *.py[cod]
 *$py.class
+src/kweb/gds_files/
+
```
Comment (sebastian-goeldi) [inline .gitignore:7]: Doesn´t  exist anymore

File: Makefile [hunk 2]
```diff
@@ -8,6 +8,11 @@ test:
 cov:
 	pytest --cov=kweb
 
+run:
+	export KWEB_FILESLOCATION=/tmp/gdsfactory
+	cd src/kweb
+	uvicorn --reload default:app
+
```
Comment (sebastian-goeldi) [inline Makefile:15]: why readd the run? The readme clearly states how to run it. Not everything needs to be a makefile. Plus I don't want to tie kweb to gdsfactory. If we do defaults like this, suddenly we won't notice in gdatasea if we don't properly unset/change a default.

**Review by sebastian-goeldi — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (sebastian-goeldi) [reply Makefile:15]: Also, this won't work under windows (admittedly the README has that issue as well)

File: .gitignore (hunk 1 above)
Comment (joamatab) [reply .gitignore:7]: sure, i can delete it

File: Makefile (hunk 2 above)
Comment (joamatab) [reply Makefile:15]: I can also remove this

**Comment by joamatab** [comment]
Hi Sebastian,

i addressed all changes, 

I would like a new release :)

**Review by sebastian-goeldi — APPROVED** [review]

**Pull request merged**